Editor/Innoactive.CreatorEditor.asmdef

```diff
@@ -12,6 +16,22 @@
     "precompiledReferences": [],
     "autoReferenced": true,
     "defineConstraints": [],
-    "versionDefines": [],
+    "versionDefines": [
+        {
+            "name": "com.unity.xr.management",
+            "expression": "",
+            "define": "UNITY_XR_MANAGEMENT"
```

Comment:
With current xr setup it felt like the defines in build settings were not removed when switching setups, is this intended? Ended up with multple different defines and a real annoying multi import thingi. Don't know if this already improved

Reply:
It is intended, those are now useless and harmless.